src/hooks/useActivityStream.ts

```diff
@@ -36,32 +43,43 @@ export const useActivityStream = (size: number) => {
   const getActivityStream = useCallback(async () => {
     try {
       const response = await api.getActivityStream(size);
-      const parsedData = JSON.parse(convert.xml2json(response, {compact: true, spaces: 2}));
-      const mappedData = parsedData.feed.entry.map((entry: ActivityStreamEntry): ActivityStreamElement => {
-        const time = getPropertyValue(entry, 'updated');
-        const icon = entry.link[1]._attributes;
-        return {
-          id: getPropertyValue(entry, 'id'),
-          time: {
-            elapsed: getElapsedTime(time),
-            value: new Date(time).toLocaleTimeString("en-US", { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' }),
-          },
-          title: decodeHtml(getPropertyValue(entry, 'title')),
-          icon: {
-            url: icon.href,
-            title: icon.title,
-          },
-          summary: decodeHtml(getPropertyValue(entry, 'summary') || ''),
-          content: decodeHtml(getPropertyValue(entry, 'content') || ''),
-        }
-      });
+      const parsedData = JSON.parse(
+        convert.xml2json(response, { compact: true, spaces: 2 }),
```

Comment:
Can we not request json via the http headers on the request? e.g. 
```
Content-Type: application/json
Accept: application/json
```

Reply:
It's an RSS feed in XML taken directly from Jira,  I think it won't be possible to get is a JSON format directly.